### PR TITLE
Recording list fixes

### PR DESCRIPTION
**Pull request by mmichael0413**
# Trello card link

# Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published

## Acceptance

https://www.notion.so/aiera/Test-Flows-Template-5203736ccfcb424c81643065f2bbf5e2

- [ ] I have reviewed the Test Flows template and it is up-to-date after considering my changes

### CODE REVIEW

File: src/components/Checkbox/index.tsx [hunk 1]
```diff
@@ -20,18 +22,31 @@ interface CheckboxUIProps extends CheckboxSharedProps {
...
+              error ? 'border-red-600 group-hover:border-red-700' : 'border-gray-300 group-hover:border-gray-400'
+          }`;
     let radius = 'rounded-sm';
     if (kind === 'radio') {
         radius = 'rounded-xl';
     }
     return (
-        <div className={`cursor-pointer flex items-center ${className} checkbox`} data-tname={name} onClick={onChange}>
+        <div
+            className={`group cursor-pointer flex items-center ${className} checkbox`}
```
Comment (getnashty) [inline src/components/Checkbox/index.tsx:38]: I dont think you're supposed to do this in tailwind... can you use the classNames util i.e 

`<div className={classNames("group cursor-pointer flex items-center checkbox", className)} ...`

File: src/components/Checkbox/index.tsx [hunk 2]
```diff
@@ -20,18 +22,31 @@ interface CheckboxUIProps extends CheckboxSharedProps {
...
+            data-tname={name}
+            onClick={onChange}
+        >
             <div className={`flex flex-shrink-0 h-4 items-center justify-center ${radius} w-4 ${checkBoxStyles}`}>
```
Comment (getnashty) [inline src/components/Checkbox/index.tsx:42]: I missed this in previous PR apparently... same comment re: how to candle variable classNames

Comment (getnashty) [reply src/components/Checkbox/index.tsx:42]: the classNames util can take unlimited args, and can use an object for conditional classes too i.e.` classNames({ 'h-10': x === 10}, var1, var2, var3, 'w-10 justify-center')`

File: src/modules/EventList/index.tsx [hunk 3]
```diff
@@ -259,17 +250,6 @@ const DefaultEventRow = ({
...
-                            <div className="flex items-center mt-[2px]">
-                                <div className="rounded-full h-[6px] w-20 bg-gray-200">
-                                    <div className={`rounded-full h-[6px] bg-blue-500 w-${hitRatioClass}`} />
-                                </div>
-                                <div className="uppercase font-semibold ml-2 text-black tracking-wide text-xs">
-                                    {numMentions || 0} hit
-                                    {numMentions !== 1 && 's'}
-                                </div>
-                            </div>
-                        )}
```
Comment (getnashty) [inline src/modules/EventList/index.tsx:272]: 😎

File: src/modules/EventList/index.tsx [hunk 4]
```diff
@@ -469,8 +449,8 @@ export const EventListUI = (props: EventListUIProps): ReactElement => {
                         </FilterBy>
                     </div>
                     <div className={classNames('flex flex-col items-center justify-center flex-1')}>
-                        {match(eventsQuery)
-                            .with({ status: 'loading' }, () => (
+                        {match(queryStatus)
```
Comment (mmichael0413) [inline src/modules/EventList/index.tsx:452]: The query statuses were originally only using the `eventsQuery` one, but we're now firing 2 queries (recent & live) so makes sense to check both.

**Review by getnashty — APPROVED** [review]

**Pull request merged**